test/JDBC/input/functions/datepart-vu-prepare.sql

```diff
@@ -43,3 +43,39 @@ CREATE PROCEDURE date_part_vu_prepare_sys_day_proc @a datetime
 AS
 SELECT DAY(@a)
 GO
+
+-- Test Case for Date Part Functions Timezone Invariance
+CREATE TABLE date_part_vu_prepare_TestDates (
+    DateID INT IDENTITY(1,1) PRIMARY KEY,
+    TestDate DATETIME
```

Comment:
We should also test for other datatypes as well - 'DATETIMEOFFSET', 'DATETIME2', 'SMALLDATETIME'